rust/driver/datafusion/tests/test_datafusion.rs

```diff
@@ -15,6 +15,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#![cfg(feature = "ffi")]
```

Comment:
Huh, ideally the tests wouldn't depend on FFI, but that can be fixed later.

Reply:
https://github.com/apache/arrow-adbc/pull/3172